stackable-base/Dockerfile

```diff
@@ -1,5 +1,6 @@
 # 8.9-1029 as of 2023-12-01
-FROM registry.access.redhat.com/ubi8/ubi-minimal:8.9@sha256:0e1879febd1be94e0d234060770ce33a543488374d91881259de9119ea75d33f
+# Manifest list digest because of multi architecture builds
```

Comment:
I'm sorry. Can you elaborate what this is?
The digest looks the same to me - structure wise - maybe a link to manifest list explanation?

Reply:
yes they are structure wise the same. 

https://www.redhat.com/architect/pull-container-image#:~:text=A%20manifest%20list%20exists%20to,system%20on%20a%20specific%20architecture).

A manifest digest is basically the same as the one of a manifest list. However, when pointing on the list instead of the simple manifest, the client chooses its architecture before downloading a corresponding blob.

Hence, building on two architectures concurrent requires the clients to decide what image to choose and pull from the list while pulling always the same digest.

Reply:
Done as a comment in Dockerfile